FE/angular/src/app/components/movie-card/movie-card.component.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { MovieCardComponent } from './movie-card.component';
+import { GoDetailsComponent } from '../navigation/go-details/go-details.component';
+
+describe('MovieCardComponent', () => {
+  let component: MovieCardComponent;
+  let fixture: ComponentFixture<MovieCardComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [MovieCardComponent, GoDetailsComponent]
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(MovieCardComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  test('should create', () => {
```

Comment:
While you tested the component instantiation, you did not test anything about the component. What about its inputs and the resulting DOM?